server.js

```diff
@@ -1,13 +1,25 @@
-app.post('/slambook', async (req, res) => {
+app.put('/slambook/:id', async (req, res) => {
+  const entryId = req.params.id;
+
   try {
-    

```

Comment:
Consider adding error handling for the 'await SlamBook.findByIdAndUpdate()' operation in case any error occurs during the update process.